worksheet_A/TerminalHacking.cpp

```diff
@@ -34,13 +44,71 @@ int main()
 	}
 
 	// Display all words
-	for each (std::string word in options)
-	{
-		std::cout << word << std::endl;
+	int i = 0;
+	int k = 0;
+
+	for each (std::string word in options){
+		std::cout << "\t" <<word << std::endl; // no matches but print it
 	}
 
 	// TODO: implement the rest of the game
+	// get user input of their chosen word
+
 
-    return 0;
+	int x = 4; //to limit 4 guesses
```

Comment:
Again, this should be e.g. `lives` rather than `x`. Generally `x` should only be used for an x-coordinate in graphics etc.